backend/src/graphql/resolver/ContributionMessageResolver.test.ts

```diff
@@ -193,53 +200,37 @@ describe('ContributionMessageResolver', () => {
           )
         })
 
-        it('calls sendAddedContributionMessageEmail', () => {
-          expect(sendAddedContributionMessageEmail).toBeCalledWith({
-            firstName: 'Bibi',
-            lastName: 'Bloxberg',
-            email: 'bibi@bloxberg.de',
-            language: 'de',
-            senderFirstName: 'Peter',
-            senderLastName: 'Lustig',
-            contributionMemo: 'Test env contribution',
-          })
+        it("don't call sendAddedContributionMessageEmail", () => {
+          expect(sendAddedContributionMessageEmail).not.toBeCalled()
         })
 
-        it('stores the ADMIN_CONTRIBUTION_MESSAGE_CREATE event in the database', async () => {
-          await expect(DbEvent.find()).resolves.toContainEqual(
+        it("don't change contribution status", async () => {
```

Comment:
single quotes? Is this accepted by the linter?
```suggestion
        it('does not change contribution status", async () => {
```